diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 2a8997c..bced5a4 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -1023,7 +1023,7 @@
 		// performance.
 		if (this.lastDeltaChangeSet && this.lastDeltaChangeSet.source === 'user') {
 			this.lastDeltaChangeSet = null;
-			var el = ReactDOM.findDOMNode(this.getEditor());
+			var el = this.getEditor().getEditorContents();
 			var editor = this.getEditor();
 			this.setEditorSelection(editor, editor.getSelection());
 		}
