diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 2338b55..0929255 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -520,7 +520,7 @@
 	},
 
 	getEditor: function getEditor() {
-		return this.quill;
+		return this.quill
 	},
 
 	getEditingArea: function getEditingArea() {
@@ -528,7 +528,7 @@
 	},
 
 	getEditingAreaRenderedNode: function getEditingAreaRenderedNode() {
-		return this.editingArea;
+		return this.editingArea.current
 	},
 
 	focus: function focus() {
